### ISSUE TITLE
Label text color for dropdown has insufficient contrast ratio

### ISSUE THREAD

**Issue by v-aaronvega** [microsoft/fluentui#20216]
### Environment Information

- **Package version(s)**: "@fluentui/react@8.36.5"
- **Browser and OS versions**: Chrome on Windows 10

### Describe the issue:

The label for a disabled drop down has an incorrect color ratio.  MAS requires a ratio of 4.5:1.

### Please provide a reproduction of the issue in a codepen:

- No need for codepen.  Issue repros in demo site.  Go to "Disabled example with defaultSelectedKey" at https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown#usage
- Use "inspect element" on the text, and open the color details.

#### Actual behavior:

Color ratio is 2.63

#### Expected behavior:

Color ratio should be 4.5+

### Documentation describing expected behavior

Suggested color: rgb(118, 118, 118)

**Comment by gouttierre** [comment]
@v-aaronvega - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows. 

@khmakoto  - Would you be able to confirm the color ratio?
cc @smhigley including you as you are working with Dropdown convergence.

**Comment by tringakrasniqi** [comment]
Thank you for filing this @v-aaronvega, although this seems to be a non-bug. According to the WCAG requirements for color contrast ratio, inactive/disabled ui components (incidental) are an exception to the 4.5:1 ratio and do not have contrast ratio requirements. Check this [link](https://w3c.github.io/wcag/guidelines/22/#contrast-minimum) for more info.
